src/models/CoinDefinition.ts

```diff
@@ -192,7 +192,7 @@ export const ALL_COINS: CoinDefinition[] = [
     name: 'Dogecoin',
     symbol: 'DOGE',
     blockchains: ['DOGE'],
-    referrals: ['pdwb-af48', '1f6v-fy9m'],
+    referrals: ['3q5x-94gx', '1f6v-fy9m'],
```

Comment:
@evan-cohen please confirm this isn't your referral code that I wiped out.

Reply:
all set - not mine.